apps/backend/src/app/app.module.ts

```diff
@@ -1,11 +1,11 @@
 import { Module } from '@nestjs/common';
 import { HotelModule } from './hotel/hotel.module';
-import { HotelController } from './hotel/hotel.controller';
-import { HotelService } from './hotel/hotel.service';
+import { ConfigModule } from '@nestjs/config';
+import cloudinaryConfig from './cloudinary/cloudinary.config';
 
 @Module({
-  imports: [HotelModule],
-  controllers: [HotelController],
-  providers: [HotelService],
+  imports: [HotelModule, ConfigModule.forRoot({ load: [cloudinaryConfig] })],
```

Comment:
consider this one, Cloudinary is feature, should not import too root: move it to cloudinary and import forFeature(() => {...})